pkg/aws_config_server/list_roles.go

```diff
@@ -219,3 +222,18 @@ func GetActiveAccountList(
 
 	return activeAccounts, nil
 }
+
+func getAcctAlias(ctx context.Context, svc iamiface.IAMAPI) (string, error) {
+	input := &iam.ListAccountAliasesInput{}
+	output, err := svc.ListAccountAliases(input)
+
+	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok && (awsErr.Code() == ignoreAWSError) {
+			logrus.WithError(err).Error("Unable to get aliases for this account")
+			return "", nil
+		}
+		return "", errors.Wrap(err, "Error getting account alias")
+	}
+
+	return *output.AccountAliases[0], nil
```

Comment:
what if this account doesn't have an alias?
```suggestion
        // no alias
        if output == nil || len(output.AccountAliases) == 0 {
            return "", nil
        }
	return *output.AccountAliases[0], nil
```